data/messages.json

```diff
@@ -110,11 +110,18 @@
     "GANGS_NOT_REGISTERED": "@error You need to register on https://sa-mp.nl/ before you can participate in a gang.",
     "GANGS_ONLINE": "{%s}[%s]{FFFFFF} - %s ({B1FC17}%d player%s{FFFFFF})",
 
+    "HOUSE_ANNOUNCE_ACCESS_CHANGED": "%s (Id:%d) has changed (%s (Id:%d))'s access to %s.",
```

Comment:
Discussed with @BeanerLVP; Will swap around name and ID so IRC logs can be checked easier.